certbot/certbot/_internal/plugins/manual.py

```diff
@@ -44,12 +44,22 @@ class Authenticator(common.Plugin):
         'remain after the current one, and $CERTBOT_ALL_DOMAINS contains a comma-separated '
         'list of all domains that are challenged for the current certificate.')
     _DNS_INSTRUCTIONS = """\
-Please deploy a DNS TXT record under the name
-{domain} with the following value:
+Please deploy a DNS TXT record with the following value:
 
 {validation}
 
-Before continuing, verify the record is deployed."""
+under the name:
```

Comment:
This is being super picky I know, but could we switch these two? At least on domains I've configured, the domain comes before the value, and it'll make just as much sense linguistically. So "under the name....with the following value".

I love the formatting of how it's presented here though, very clear, nice work!

Reply:
Sure :) I just followed the order of the HTTP-01 instructions.